### PR TITLE
James Palmer

### PR DESCRIPTION
**Pull request by jpalmerr**
Basic challenge

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
+require 'sinatra/base'
+
```
Comment (hybridbad) [inline app.rb]: require './lib/turn'

Comment (hybridbad) [reply app.rb]: whoops wrong button meant to comment

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+class RPS < Sinatra::Base
+  get '/' do
+    erb :index
+  end
+
+  post '/names' do
+    @name = params[:name]
+    erb :play
+  end
+
```
Comment (hybridbad) [inline app.rb]: inject into instance on your play POST route (well global var but I don't know how do a singleton class yet)

in names
@player_1 = params[:name] 

in play
$turn = Turn.new(player_1, player_2)
@move = params[:move]

in result
@turn = $turn

@turn.game(@move)

File: README.md [hunk 3]
```diff
@@ -1,23 +1,5 @@
 # RPS Challenge
```
Comment (hybridbad) [inline README.md]: This readme doesn't really tell me much about 'your' program only the instructions of the challenge. Maybe add some instructions for people so they know how to run it. Break down the user stories and explain your approach?

Comment (hybridbad) [reply README.md]: Sorry I am blind AF.

Saw your process.md file - maybe export that into the README as thats the first thing that gets loaded

File: app.rb [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+# how can i move these to a ruby file within lib directory? => i can test them
+
+  def computer_move
+    ['Rock', 'Paper', 'Scissors'].sample
+  end
+
+  def game(user = @move, computer = @computer_move)
+
```
Comment (hybridbad) [inline app.rb]: nice use of case switch statement

File: process.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+After submitting their move, the user is directed to the /play page.
+At this point, the page simply returns what move the user entered.
+
+###### Ideas
+
+- a 'play again' link returning the user to the /names page
+
+###### Difficulties
+
```
Comment (hybridbad) [inline process.md]: Good reflection!

File: spec/features/web_helpers.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+def sign_in_and_play
+  visit('/')
+  fill_in :name, with: 'James'
+  click_button 'Lets play!'
+end
+
```
Comment (hybridbad) [inline spec/features/web_helpers.rb]: good use of web helpers, using things learnt from battle challenge

File: public/stylesheet.css [hunk 7]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (hybridbad) [inline public/stylesheet.css]: nice to see some css practise! Have you thought about using a css library like bootstrap?

File: app.rb (hunk 2 above)
Comment (jpalmerr) [reply app.rb]: This is what I was trying but think I went wrong somewhere, probably got a detail wrong along the way. May try to refactor if time allows. Thankyou

File: app.rb (hunk 1 above)
Comment (alexanderchen6) [reply app.rb]: yes. agreee need add turn here

**Pull request closed without merging**